Dockerfile

```diff
@@ -1,31 +1,77 @@
-FROM php:7.2-fpm
+FROM alpine:latest AS builder
```

Comment:
I would rather stay with Debian to avoid strange bugs which are caused by changing to alpine.